sonar/modules/documents/marshmallow/json.py

```diff
@@ -100,6 +100,9 @@ class DocumentMetadataSchemaV1(StrictKeysMixin):
     projects = fields.List(fields.Dict())
     oa_status = SanitizedUnicode()
     hiddenFromPublic = fields.Boolean()
+    customField1 = fields.String()
```

Comment:
You can add a `minLength` constraint to be consistent with the JSONSchema.

Reply:
Done, but it's a bit overkill to do the validation twice...